ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -1562,6 +1564,33 @@ def verify_consumer_storagecluster(sc_data):
     assert re.match("\\d+[PT]", granted_capacity)
 
 
+def verify_managedocs_security():
+    """
+    Check ocs-osd-deployer-operator permissions:
+    1. Verify `runAsUser` is not 0
+    2. Verify `SecurityContext.allowPrivilegeEscalation` is set to false
+    3. Verify `SecurityContext.capabilities.drop` contains ALL
+    """
+    pod_obj = OCP(
+        kind="pod",
+        namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+        selector=constants.MANAGED_CONTROLLER_LABEL,
+    )
+    deployer_yaml = pod_obj.get().get("items")[0]
+    containers = deployer_yaml["spec"]["containers"]
+    for container in containers:
+        log.info(f"Checking container {container['name']}")
+        userid = container["securityContext"]["runAsUser"]
+        log.info(f"runAsUser is {userid}. Verifying it is not 0")
+        assert userid > 0
```

Comment:
It is a good practice to not include assert statements into module code and use exceptions instead. Asserts should be only in tests. In this case the function is used to run checks so it might be ok.

Reply:
Assert statements are used everywhere in ocs_install_verification function, so I wanted to keep things consistent. If we want to change it, I suggest creating an issue.